doc/faq.md

```diff
@@ -85,24 +85,28 @@ DPVS side TOA format is defined in `include/ipvs/proto_tcp.h`, while option code
   };
   ... ...
   struct tcpopt_addr {
-     uint8_t opcode;
-     uint8_t opsize;
-     uint16_t port;
-     uint32_t addr;
+      uint8_t opcode;
+      uint8_t opsize;
+      __be16 port;
+      uint8_t addr[16];
   } __attribute__((__packed__));
   ```
 
+> In case of nat64, RS's codes need a little changes to get real client IP/port. DPVS provide an example TCP server and a nginx patch for nat64 toa. Pls refer to [example_nat64](../kmod/toa/example_nat64).
```

Comment:
In case of nat64, RS's codes need a **few** changes to get **the** real **client's** IP/port. 
**Please** refer to